docs/python/editing.md

```diff
@@ -86,20 +86,38 @@ While editing, you can right-click different identifiers to take advantage of se
 
 ## Quick Fixes
 
+### Add Missing Imports
+
 The add imports Quick Fix when using Pylance allows you to quickly complete import statements. First, begin by typing a package name within the editor. You will notice a Code Action is available to automatically complete the line of source code (as long as you have the module installed within the environment). Hover over the text (marked with a squiggle) and then select the Code Action light bulb when it appears. You can then select from a list of potential imports.
-![Adding an import](images/editing/quickFix.gif)
+
+![Add import code action](images/editing/quickFix.png)
 
 This Code Action also recognizes some of the popular abbreviations for the following common Python packages: `numpy` as np, `tensorflow` as tf, `pandas` as pd, `matplotlib.pyplot` as plt, `matplotlib` as mpl, `math` as m, `scipi.io` as spio, and `scipy` as sp, `panel` as pn, and `holoviews` as hv.
 
 ![Common package abbreviations](images/editing/packageAbbreviations.gif)
 
-The import suggestions list is ordered with import statements for packages (or modules) at the top. It will also include statements for more modules and/or members (classes, objects, etc.) from specified packages.
+The import suggestions list displays  the top 3 high-confidence import options, prioritized based on: most recently used imports, symbols from the same module, symbols from the standard library, symbols from user modules, symbols from third-party packages, and finally sorting by module and symbol name.  In the case where the 3 high-confidence import options aren’t what you are looking for, Pylance has a **Search for additional import matches** Code Action, which displays a quick pick menu that allows you to search for import options that prefix-match the missing import symbol.
+
+![Search for additional import matches Code Action](images/editing/search-imports-code-action.gif)
+
+Pylance also offers a **Change spelling** Code Action, which displays import suggestions for missing imports due to typos.
+
+![Change spelling code action on missing import due to a typo](images/editing/change-spelling-code-action.gif)
+
+
+Just like with auto imports, only top-levels symbols are suggested by default. You can customize this behavior through the `python.analysis.packageIndexDepths` setting. Note that user symbol aliases (e.g. `import foo as bar`) are not supported by these Code Actions.
```

Comment:
>  Note that user symbol aliases (e.g. `import foo as bar`) are not supported by these Code Actions.

I'm not sure what this means. Is it referring to the popular abbreviation list (ex. pandas as pd) mentioned above? Or is it saying that it doesn't support adding a custom alias to the import statement it generates? Or something else?

Reply:
what it trying to say is when there is

```
module/__init__.py
from module.impl import symbol as symbol

module/impl.py
class symbol: ...
```

when module is 3rd party library, we will show `symbol` from `module/__init__` in `add/auto import`, but when module is user module/file, we will show `symbol` from `module/impl`.

in another word, we don't support `alias symbol` from `user file/module` in `add/auto import`

Reply:
I think the current phrasing is going to confuse users, but I'm not sure how to make it more clear. Maybe this isn't important enough to mention here?

I'm also wondering if this is the right place to mention it, since this paragraph is about indexing depth but that isn't an issue with user code, is it?

Reply:
ya, that's for 3rd party packages. 

but also, I believe we want to add somewhere that we currently don't support alias symbol from user files since that question comes up again and again from users.

Reply:
I think the confusion may be because it was brought up after the indexing depth setting, so it felt like they were talking about similar issues when in fact the paragraph is just acknowledging some limitations? Coming from that assumption, I just flipped the order of the sentence and put it all under a "note" block to try to clarify that